docs/design/contributing.md

```diff
@@ -0,0 +1,5 @@
+## Coding Style
```

Comment:
Keep the file name in capital e.g. CONTRIBUTING.md

Reply:
done